readme.md

```diff
@@ -10,13 +10,17 @@ This package contains scripts that help build client code:
 
 - `inline-environment-variables` replaces references to `process.env.<envVarName>` with their values in a JS file.
 
-# Installation
+---
```

Comment:
I'm not a fan of manually styling md's and would rather the md renderer do it's thing. Not only because I trust the renderer more than myself to decide when to add lines for best readability, but also because manually doing this could lead to consistency issues.